reana_job_controller/kubernetes_job_manager.py

```diff
@@ -191,3 +201,42 @@ def add_shared_volume(self, job):
         job['spec']['template']['spec']['containers'][0][
             'volumeMounts'].append(volume_mount)
         job['spec']['template']['spec']['volumes'].append(volume)
+
+    def _add_krb5_sidecar_container(self, job, secrets_volume_mount):
+        """Add  sidecar container for a job."""
+        krb5_config_map_name = 'krb5-conf'
+        ticket_cache_volume = {
+            'name': 'krb5-cache',
+            'emptyDir': {}
+        }
+        krb5_config_volume = {
+            'name': 'krb5-conf',
+            'configMap': {'name': krb5_config_map_name}
+        }
+        volume_mounts = [
+            {
+                'name': ticket_cache_volume['name'],
+                'mountPath': current_app.config['KRB5_TOKEN_CACHE_LOCATION']
+            },
+            {
+                'name': krb5_config_volume['name'],
+                'mountPath': '/etc/krb5.conf',
+                'subPath': 'krb5.conf'
+            }
+        ]
+        keytab_file = os.environ.get('HTCONDORCERN_KEYTAB')
```

Comment:
Both variables username and keytab should be generic and not depend on HTCondor as Kerberos might be used outside HTCondor context.

The problem is that [here](https://github.com/reanahub/reana-workflow-controller/blob/5e31c1536db56898542c1d9abca42451a0507d93/reana_workflow_controller/workflow_run_manager.py#L374) we get the HTCondor dependent secret name, and then we inject it in RJC with a generic name (`CERN_USER`), which is correct. The solution is to amend documentation so we tell users that they have to set `CERN_USER_NAME` and `CERN_USER_KEYTAB` instead of the `HTCONDOR*` which we currently have, so in the code, we will depend on them:
```diff
- secrets_store.get_secret_value('HTCONDORCERN_USERNAME') 
+ secrets_store.get_secret_value('CERN_USER_NAME') 
- secrets_store.get_secret_value('HTCONDORCERN_KEYTAB') 
+ secrets_store.get_secret_value('CERN_USER_KEYTAB')
```